DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -298,12 +298,13 @@ export class SearchSortFilterAndPaginate {
   private startLoadingSpinner(): void {
     this.spinnerContainer?.classList.remove('display-none');
     this.spinner?.classList.remove('loading-spinner');
-    this.areaToHide?.classList.add('display-none');
   }
 
   private stopLoadingSpinner(): void {
     this.spinnerContainer?.classList.add('display-none');
     this.spinner?.classList.add('loading-spinner');
-    this.areaToHide?.classList.remove('display-none');
+    if (this.areaToHide !== null) {
+      this.areaToHide.id = '';
+    }

```

Comment:
Is there a reason why we're doing this by removing an id rather than changing the style associated with that id?

Reply:
No good reason. I've changed this to set the style instead.